charts/flyte-core/values.yaml

```diff
@@ -466,6 +466,8 @@ storage:
     region: us-east-1
     # -- type of authentication to use for S3 buckets, can either be iam or accesskey
     authType: iam
+    # -- Extra arguments, as a JSON object of key/value pairs, passed to S3 upload
+    extraArgs: ""
```

Comment:
(somehow) need to populate this field with the right values e.g. awskmsid like `arn:aws:kms:us-west-2:...`